client/Elements/Components/Topics/TopicsList.patternfly.tsx

```diff
@@ -79,7 +79,7 @@ export const TopicsList: React.FunctionComponent<ITopicList> = ({
   ];
   const rowData =
     filteredTopics?.topics.map((topic) => [
-      topic?.name,
+      { title: <a href={'#/topic/' + topic?.name}>{topic?.name}</a> },
```

Comment:
Can we use template strings instead of concatenation for passing down values to html attributes, will look better
```
<a href={`#/topic/${topic?.name}`}>{topic?.name}</a>
<BreadcrumbItem to={`#/topic/${topicName}`}>{topicName}</BreadcrumbItem>
```